src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -103,10 +103,10 @@ public Map<CreationEnum, List<? extends AzureAccount>> addAzureAccounts(
                 User userAzure = azureUserService.getUser(azureAccount.getEmail());
 
                 if (userAzure != null
```

Comment:
This process is also used for Admin's too (so will have firstname and surname in two seperate fields, rather than just in the given name).

I think we need to check with Chris whether:

1) Should we sent the duplicate email for Admin users too?
2) And if so, would we want their full name displayed?

Reply:
As discussed, it is fixed.